htmd/parameterization/cli.py

```diff
@@ -512,8 +528,13 @@ def _output_results(mol, parameters, original_coords, args):
 
     logger.info('=== Results ===')
 
+    dir = os.path.join(args.outdir, 'parameters', args.forcefield)
```

Comment:
`dir` --> `paramDir`. It shadow built-in `dir`

Reply:
Correct!